tests/foreman/ui/test_ansible.py

```diff
@@ -115,3 +118,65 @@ def test_positive_create_variable_with_overrides(target_sat):
             }
         )
         assert session.ansiblevariables.search(key)[0]['Name'] == key
+
+
+@pytest.mark.skip_if_open("BZ:2155814")
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([8, 9])
+def test_positive_config_report_ansible(session, target_sat, module_org, rhel_contenthost):
+    """
+    Test Config Report generation with Ansible Jobs.
+
+    :id: 118e25e5-409e-44ba-b908-217da9722576
+
+    :steps:
+        1. Register a content host with satellite
+        2. Import a role into satellite
+        3. Assign that role to a host
+        4. Assert that the role was assigned to the host successfully
+        5. Run the Ansible playbook associated with that role
+        6. Check if the report is created successfully
+
+    :expectedresults:
+        1. Host should be assigned the proper role.
+        2. Job report should be created.
+
+    :CaseAutomation: Automated
```

Comment:
CaseAutomation already defined at module level

Reply:
At the test level, it helps in bifurcating the manual vs automated scenarios

Reply:
yes, but if something is defined at module level so we don't repeat them at test level, so this can be removed